result.json

```diff
@@ -0,0 +1,15775 @@
+
+{
+    "address_components": [
+        {
+            "long_name": "375",
+            "short_name": "375",
+            "types": [
+                "street_number"
+            ]
+        },
+        {
+            "long_name": "11th St",
+            "short_name": "11th St",
+            "types": [
+                "route"
+            ]
+        },
+        {
+            "long_name": "South of Market",
+            "short_name": "South of Market",
+            "types": [
+                "neighborhood",
+                "political"
+            ]
+        },
+        {
+            "long_name": "San Francisco",
+            "short_name": "SF",
+            "types": [
+                "locality",
+                "political"
+            ]
+        },
+        {
+            "long_name": "San Francisco County",
+            "short_name": "San Francisco County",
+            "types": [
+                "administrative_area_level_2",
+                "political"
+            ]
+        },
+        {
+            "long_name": "California",
+            "short_name": "CA",
+            "types": [
+                "administrative_area_level_1",
+                "political"
+            ]
+        },
+        {
+            "long_name": "United States",
+            "short_name": "US",
+            "types": [
+                "country",
+                "political"
+            ]
+        },
+        {
+            "long_name": "94103",
+            "short_name": "94103",
+            "types": [
+                "postal_code"
+            ]
+        }
+    ],
+    "adr_address": "<span class=\"street-address\">375 11th St</span>, <span class=\"locality\">San Francisco</span>, <span class=\"region\">CA</span> <span class=\"postal-code\">94103</span>, <span class=\"country-name\">USA</span>",
+    "formatted_address": "375 11th St, San Francisco, CA 94103, USA",
+    "formatted_phone_number": "(415) 626-1409",
+    "geometry": {
+        "location": {
+            "lat": 37.7710938,
+            "lng": -122.4126502
+        },
+        "viewport": {
+            "northeast": {
+                "lat": 37.7724427802915,
+                "lng": -122.4113012197085
+            },
+            "southwest": {
+                "lat": 37.76974481970851,
+                "lng": -122.4139991802915
+            }
+        }
+    },
+    "icon": "https://maps.gstatic.com/mapfiles/place_api/icons/bar-71.png",
+    "id": "ea6ff166f8491ff9efed520ae1c49ed97e93675a",
+    "international_phone_number": "+1 415-626-1409",
+    "name": "DNA Lounge",
+    "opening_hours": {
+        "open_now": true,
```

Comment:
Is open_now a dynamic value considering the current time, open and close time?

Reply:
Yes. At submit time, it was opened. I think opening_hours is calculated by us,,,,

Reply:
It sounds open_now is valueless... :(

Reply:
yes. this json data is just test data and whole data of place. we need to extract important data.